agent/agent.go

```diff
@@ -218,3 +228,26 @@ func initializeStateManager(cfg *config.Config, taskEngine engine.TaskEngine, cl
 	}
 	return stateManager, nil
 }
+
+func printVersion() {
```

Comment:
Can you move this out of the main `agent.go` file and into the `version` package?  Might want to inject the endpoint or even a fully constructed `dockerClient`, but I'd rather that we start slimming this file down than add new things to it.